week-2/13-order/readme.md

```diff
@@ -0,0 +1,13 @@
+# Item order
+
+The `flex-order` property is used to move a *specific* item to a different position in the list.
+
+> **Remember:** if you forgot how to use the `flex-order` property, you can always:
+> 1. Google the word "flexbox"
+> 2. Click on the link from "CSS-tricks"
+> ![Screenshot of how to google flexbox](/images/13-googling-flexbox.png)
+
+Exercise:
+
+1. Open up the HTML page for this exercise in Chrome
```

Comment:
By this point, the students should be comfortable running `npm run X`, which will open the page automatically. It might be confusing if we tell them to open the HTML page in Chrome -- they might think we mean actually opening the file directly.

Suggested alternative:

> 1. Look at the First list of countries. Then look at the First Result displayed at the bottom. Find the differences between the two lists.
> 2. Use the `order` property to make the First list of countries look like the First Result list.
> 3. Do the same thing for the Second and Third list.